src/alt.cc

```diff
@@ -2437,7 +2437,28 @@ void Alt::Simple::init_multi_ys() {
   m_ys = Yield::Multi(tracks_);
 
   if (is_terminal()) {
-    assert(tracks_ == 1);
+	assert(tracks_ == 1);
+
+	// a terminal like CHAR or ROPE can have one argument, that is an implicit filter like
+	// CHAR('a') or ROPE("kurt"), which shall only accept sub-words that are 'a' or 'kurt', respectively
```

Comment:
you chose ROPE("stefan") in the other files as an example - I'd change the "kurt" ones into "stefan" for the sake of consistency